include/circt/Dialect/Moore/MooreOps.td

```diff
@@ -146,7 +146,40 @@ def VariableOp : MooreOp<"variable", [
   let arguments = (ins StrAttr:$name, Optional<UnpackedType>:$initial);
   let results = (outs Res<UnpackedType, "", [MemAlloc]>:$result);
   let assemblyFormat = [{
-    `` custom<ImplicitSSAName>($name) ($initial^)? attr-dict
+    custom<ImplicitSSAName>($name) ($initial^)? attr-dict
```

Comment:
I think the `` is necessary here as a hint to the MLIR printer that it should not insert a space here. (ImplicitSSAName already adds a space.) Otherwise you get a double-space after the operation name, like `moore.var  name "foo" ...`. Same below for the NetOp.